index.js

```diff
@@ -1,6 +1,25 @@
 /* jshint node: true */
 'use strict';
 
+var path = require('path');
+var mergeTrees = require('broccoli-merge-trees');
+
 module.exports = {
-  name: 'lodash'
+  name: 'lodash',
+
+  treeForAddon: function(tree) {
+    var lodashPath = path.join(this.project.root, 'node_modules', 'ember-lodash', 'node_modules', 'lodash-es');
```

Comment:
We can't `require.resolve` here unfortunately.  This is the best I got for determining the path.  Should be fine though.